test/quantization/fx/test_equalize_fx.py

```diff
@@ -238,3 +239,64 @@ def forward(self, x):
             m = M().eval()
             prepared = prepare_fx(m, qconfig_dict, equalization_qconfig_dict=equalization_qconfig_dict)
             self.checkGraphModuleNodes(prepared, expected_node_occurrence=node_occurrence)
+
+    def test_input_weight_equalization_convert(self):
```

Comment:
Can we extend this test/add a new test that verifies the graph structure after equalization is done?

Reply:
Would this be after the equalization functions or after all of convert?

Reply:
I think we can do both - check the output of `_convert_equalization_ref` and `convert_fx`

Reply:
this was broken on macos
```
Traceback (most recent call last):
  File "/Users/distiller/project/test/quantization/fx/test_equalize_fx.py", line 294, in test_input_weight_equalization_convert
    convert_fx(prepared)  # Check if compile?
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/quantization/quantize_fx.py", line 543, in convert_fx
    return _convert_fx(graph_module, is_reference, convert_custom_config_dict, _remove_qconfig=_remove_qconfig)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/quantization/quantize_fx.py", line 477, in _convert_fx
    is_standalone_module, _remove_qconfig_flag=_remove_qconfig)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/quantization/fx/convert.py", line 446, in convert
    convert_custom_config_dict=convert_custom_config_dict)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/quantization/fx/quantization_patterns.py", line 687, in convert
    quantized = qlinear.from_float(self.linear)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/nn/quantized/modules/linear.py", line 276, in from_float
    dtype=dtype)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/nn/quantized/modules/linear.py", line 151, in __init__
    self._packed_params = LinearPackedParams(dtype)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/nn/quantized/modules/linear.py", line 19, in __init__
    self.set_weight_bias(wq, None)
  File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/nn/quantized/modules/linear.py", line 24, in set_weight_bias
    self._packed_params = torch.ops.quantized.linear_prepack(weight, bias)
Didn't find engine for operation quantized::linear_prepack NoQEngine
```